index.js

```diff
@@ -386,6 +391,81 @@ exports.getLastThreadPost = function (thread) {
   return msg
 }
 
+/* post revision utils */
+
+exports.getRevisions = function(ssb, thread, callback) {
+  function collectRevisions(thread) {
+    // this function walks the revisions of a given thread, collecting them up
+    // asyncly
+
+    var threadRevisions = thread.related
+        .map(function(relatedMsg) {
+      
+        if (relatedMsg.value.content.type === 'post-edit' &&
```

Comment:
in situations like this, I'll usually assign content to a variable so I dont have to have giant references like that. Eg:

```js
var c = relatedMsg.value.content
if (c.type == 'post-edit' && c.revision && ...)
```